gitlab-runner-17.2.yaml

```diff
@@ -45,27 +45,36 @@ subpackages:
           output: gitlab-runner-helper
           ldflags: -w -X gitlab.com/gitlab-org/gitlab-runner/common.NAME=${{package.name}} -X gitlab.com/gitlab-org/gitlab-runner/common.VERSION=v${{package.version}}
 
-  - name: "gitlab-runner-compat-${{vars.short-package-version}}"
-    description: "Gitlab-runner Compat package for alpine"
+  - name: "gitlab-runner-oci-entrypoint-${{vars.short-package-version}}"
+    description: "Gitlab-runner oci entrypoint for alpine"
```

Comment:
Can we update this description? Not sure why we are talking about another distro here (alpine)